site/class-2-style/index.css

```diff
@@ -0,0 +1,29 @@
+html {
```

Comment:
You’re uploading files to our `site/class-2-style/` directory, but it should go in `site/class-2-style/senmetsu`.